libkbfs/tlf_journal.go

```diff
@@ -1037,6 +1141,31 @@ func (j *tlfJournal) archiveBlockReferences(
 	return nil
 }
 
+// convertImmutableBareRMDToRMD decrypts the MD in the given bare root
```

Comment:
I think this should return an `ImmutableRootMetadata`; looks like the callers just do the conversion anyway, and a hypothetical caller that modifies the returned `RMD` would also modify the ibrmd.